src/org/openbravo/service/system/SystemService.java

```diff
@@ -467,6 +468,8 @@ private void enableConstraints(Platform platform) {
       platform.enableAllTriggers(con, xmlModel, false);
       log4j.info("   Enabling foreign keys");
       platform.enableAllFK(con, xmlModel, false);
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
It's ok to only print the stack trace? We are not rethrowing this exception nor handling it?